ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/PutBulk.java

```diff
@@ -102,6 +134,42 @@ public PutBulkResult call() throws Exception {
         return new PutBulkResult("SUCCESS: Wrote all the files in " + this.inputDirectory.toString() + " to bucket " + this.bucketName);
     }
 
+    private ArrayList<Ds3Object> getDs3Objects(final Iterable<Path> filteredObjects) throws IOException {
```

Comment:
In general I would use `ImmutableList`, and at the very least `List` instead of `ArrayList`